support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -28,14 +32,20 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     context: Context,
     services: Services
   ): FutureHandlerResult = {
+    val now = () => OffsetDateTime.now
```

Comment:
as a rule I prefer to pass in the side effects, to make the tests safer and the code easier to reason about